views/js/runner/ui/toolbox/menu.js

```diff
@@ -432,6 +445,9 @@ define([
                 this.$menuButton.off('.menuNavigation');
             });
 
+        // Apply default properties to the menuComponent
+        _.defaults(menuComponent, _defaults);
```

Comment:
I'm going to opt to use default properties. I prefer this to adding conditionals around all menu properties.